test/index.test.ts

```diff
@@ -3188,6 +3188,94 @@ describe('Unit | Utility | changeset', () => {
     expect(dummyChangeset.isValid).toEqual(true);
   });
 
+  /**
+   * #removeError
+   */
+
+  it('#removeError removes an error from the changeset', () => {
+    let dummyChangeset = Changeset(dummyModel);
+    dummyChangeset.addError('email', {
+      value: 'jim@bob.com',
+      validation: 'Email already taken'
+    });
+
+    expect(dummyChangeset.isInvalid).toEqual(true);
+    expect(get(dummyChangeset, 'error.email.validation')).toBe('Email already taken');
+
+    dummyChangeset.removeError('email');
+    expect(dummyChangeset.isValid).toEqual(true);
+
+    expect(get(dummyChangeset, 'error.email.validation')).toBe(undefined);
+  });
+
+  it('#removeError using an invalid key does not throw an error', () => {
+    let dummyChangeset = Changeset(dummyModel);
+    dummyChangeset.addError('email', {
+      value: 'jim@bob.com',
+      validation: 'Email already taken'
+    });
+
+    expect(dummyChangeset.isInvalid).toEqual(true);
+    expect(get(dummyChangeset, 'error.email.validation')).toBe('Email already taken');
+
+    dummyChangeset.removeError('email');
+    dummyChangeset.removeError('foo');
+    expect(dummyChangeset.isValid).toEqual(true);
+
+    expect(get(dummyChangeset, 'error.email.validation')).toBe(undefined);
+  });
+
+  it('#removeError removing one error leaves the other', () => {
+    let dummyChangeset = Changeset(dummyModel);
+    dummyChangeset.addError('email', {
+      value: 'jim@bob.com',
+      validation: 'Email already taken'
+    });
+    dummyChangeset.addError('age', {
+      value: '0',
+      validation: 'Age is too low'
+    });
+
+    expect(dummyChangeset.isInvalid).toEqual(true);
+    expect(get(dummyChangeset, 'error.email.validation')).toBe('Email already taken');
+    expect(get(dummyChangeset, 'error.age.validation')).toBe('Age is too low');
+
+    dummyChangeset.removeError('email');
+    expect(dummyChangeset.isValid).toEqual(false);
+
+    expect(get(dummyChangeset, 'error.email.validation')).toBe(undefined);
+    expect(get(dummyChangeset, 'error.age.validation')).toBe('Age is too low');
+  });
+
+  /**
+   * #removeErrors
+   */
+
+  it('#removeErrors removes all errors', () => {
+    let dummyChangeset = Changeset(dummyModel);
+    dummyChangeset.addError('email', {
+      value: 'jim@bob.com',
+      validation: 'Email already taken'
+    });
```

Comment:
Could you add multiple errors, and test that they are all gone?

Reply:
@knownasilya Done